sys/linux/fs_ioctl_f2fs.txt

```diff
@@ -25,6 +25,10 @@ ioctl$F2FS_IOC_GET_COMPRESS_BLOCKS(fd fd, cmd const[F2FS_IOC_GET_COMPRESS_BLOCKS
 ioctl$F2FS_IOC_RELEASE_COMPRESS_BLOCKS(fd fd, cmd const[F2FS_IOC_RELEASE_COMPRESS_BLOCKS], arg ptr[out, int64])
 ioctl$F2FS_IOC_RESERVE_COMPRESS_BLOCKS(fd fd, cmd const[F2FS_IOC_RESERVE_COMPRESS_BLOCKS], arg ptr[out, int64])
 
+# F2FS_IOC_SHUTDOWN on root fs effectively brings the machine down in weird ways.
+# Fortunately, the value does not conflict with any other ioctl commands for now.
+ioctl$F2FS_IOC_SHUTDOWN(fd fd, cmd const[F2FS_IOC_SHUTDOWN], args ptr[in, flags[f2fs_shutdown_flag, int32]]) (disabled)
```

Comment:
What's the point of adding the ioctl if it's disabled?

Reply:
It looks like it's missing otherwise.  I think it's better to explicitly list it.

Reply:
Eric said that we should still have it so no one think it's missing.

Reply:
That, but also the plan was to use disabled syscalls to replace this code:
https://github.com/google/syzkaller/blob/master/sys/linux/init.go#L233-L266
and prevent the fuzzer from morphing other ioclt's into the disabled ioctl's.